sympy/calculus/util.py

```diff
@@ -1129,7 +1129,14 @@ def __add__(self, other):
                     other is S.NegativeInfinity and self.max is S.Infinity:
                 return AccumBounds(-oo, oo)
             elif other.is_extended_real:
-                return AccumBounds(Add(self.min, other), Add(self.max, other))
+                if self.min is S.NegativeInfinity and self.max is S.Infinity:
+                    return AccumBounds(-oo, oo)
+                elif self.min is S.NegativeInfinity:
+                    return AccumBounds(-oo,self.max+other)
```

Comment:
Please add a space after comma and either side of +, and on the next return line as well.